internal/helm/helm.go

```diff
@@ -27,26 +28,41 @@ var (
 func FetchChart(repoURL, name, version, path, directory string) error {
 	getters := getter.All(settings)
 
-	url, err := repo.FindChartInRepoURL(
-		repoURL,
-		name,
-		version,
-		"", "", "", getters,
-	)
-	if err != nil {
-		return err
-	}
+	var chartPath string
+	var err error
 
-	dl := downloader.ChartDownloader{
-		Out: os.Stderr,
-		// RepositoryConfig: repoConfig,
-		// RepositoryCache:  repoCache,
-		Getters: getters,
-	}
+	if strings.HasPrefix(repoURL, "oci://") {
+		// OCI support
+		ref := fmt.Sprintf("%s/%s:%s", repoURL, name, version)
+		dl := downloader.ChartDownloader{
+			Out:     os.Stderr,
+			Getters: getters,
+		}
+		chartPath, _, err = dl.DownloadTo(ref, "", path)
```

Comment:
Does the code work fine if you have a version in here (does it ignore it with OCI?)  If so, can we just make it the same code?